packages/ag-charts-website/src/content/docs/linear-gauge/index.mdoc

```diff
@@ -50,6 +52,8 @@ return (
 
 {% if isFramework("angular") %}
 
+To create a Linear Gauge, use the `<ag-gauge />` template with the type `linear-gauge`.
```

Comment:
template / import? not sure what to call it in angular

Reply:
```suggestion
To create a Linear Gauge, use the `<ag-gauge />` component with the type `linear-gauge`.
```